streams/shared/src/main/scala/zio/stream/experimental/ZPipeline.scala

```diff
@@ -397,6 +397,15 @@ object ZPipeline {
         stream.mapZIO(f)
     }
 
+  /**
+   * Emits the provided chunk before emitting any other value.
+   */
+  def prepend[A](values: Chunk[A]): ZPipeline[Any, Nothing, A, A] =
+    new ZPipeline[Any, Nothing, A, A] {
+      def apply[R, E](stream: ZStream[R, E, A])(implicit trace: ZTraceElement): ZStream[R, E, A] =
+        ZStream.fromChunk(values) ++ stream
+    }
+
```

Comment:
We could add `branchAfter` that just calls the implemnetation on the stream here too - if I understood @adamgfraser 's recommendation :)

Reply:
Oh, I forgot to do that 🤦

Reply:
Done